interview-questions.md

```diff
@@ -9,59 +9,62 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: The term "this" is used when youre calling on a variable that is nesting within a component. Kinda like calling within it self. 
 
-  Researched answer:
+  Researched answer: after some research, im even more confused 
 
 
 
-2. What is React? Why would you use it?
+2. What is React? Why would you use it? 
 
-  Your answer:
+  Your answer: React is a prepackaged framework for building UI. It has a unique attribute of being able to reload components within the site independently. I think it also has the ability to dynamically be able to react to the screen accessing the site and automatically fit to screen 
 
   Researched answer:
+  React. js is an open-source JavaScript library that is used for building user interfaces it is extremely flexible offers a great developer experience has wonderful support and resources and is easy to test
 
 
 
 3. Which lifecycle method is required in a React class component?
 
-  Your answer:
+  Your answer: extended?
 
-  Researched answer:
+  Researched answer: Render is the only required method in a class component. 
 
 
 
 4. What is JSX? What is one notable difference between HTML and JSX?
 
-  Your answer:
+  Your answer: JSX is the hybrid of HTML and Javascript. It allows for Javascript to be inbeded into html code directly, 
 
-  Researched answer:
+  Researched answer: JSX allows you to use HTML tags inside of React elements
 
 
 
-5. What is yarn? What file(s) are modified in a React application when you run yarn?
+5. What is yarn? What file(s) are modified in a React application when you run yarn? 
 
-  Your answer:
+  Your answer: Yarn is a environment that holds programing packages that run react and others.
 
-  Researched answer:
+  Researched answer: Yarn is a package manager. I dont understand that last part.
```

Comment:
Recall when with use yarn commands how several files were added to your projects. Yarn  adds all the necessary code snippets and software to get projects running. It's kind of like someone providing all the DJ equipment, power, stage, everything setup and ready to go, then all you have to bring is your energy and skills to party. Yarn controls the package that is code/software necessary for your project so you don't have to go research and download over 15,000 pieces of code to get rolling. Thank you to whoever took the time to group that stuff.